plugin/http/httptrace/httptrace.go

```diff
@@ -27,6 +27,18 @@ import (
 
 // TODO(jbd): Add godoc examples.
 
+// Attributes recorded on the span for the requests.
+const (
+	URL          = "http.url"
+	Host         = "http.host"
+	Method       = "http.method"
+	Path         = "http.path"
+	UserAgent    = "http.user_agent"
+	StatusCode   = "http.status"
```

Comment:
Where are we getting the enumeration of these attributes?

Reply:
I mention this because "http.status_code", "http.status" could even be different according to Go's HTTP API